### PR TITLE
HDDS-6792: [Merge RocksDB in Datanode] Fix issues reportd by sonar.

### PR DESCRIPTION
**Pull request by ChenSammi**
https://issues.apache.org/jira/browse/HDDS-6792 

Sonar reported issues:

https://sonarcloud.io/project/issues?resolved=false&types=BUG&branch=HDDS-3630&sinceLeakPeriod=true&severities=BLOCKER%2CCRITICAL%2CMAJOR%2CMINOR&id=hadoop-ozone

### CODE REVIEW

**Comment by ChenSammi** [comment]
cc @guihecheng

File: hadoop-ozone/dist/src/main/compose/compatibility/test.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ export COMPOSE_DIR
 
 export SECURITY_ENABLED=false
 export OZONE_REPLICATION_FACTOR=1
-
+HDDS-6792
```
Comment (guihecheng) [inline hadoop-ozone/dist/src/main/compose/compatibility/test.sh:23]: I'm not sure whether it is intended or not, I didn't found other similar usages, or just a typo?

**Review by guihecheng — COMMENTED** [review]
Most changes looks good, and these fixes should solve the sonar reported issues from my view, let's see if it works.
Only a tiny question above.

File: hadoop-ozone/dist/src/main/compose/compatibility/test.sh (hunk 1 above)
Comment (ChenSammi) [reply hadoop-ozone/dist/src/main/compose/compatibility/test.sh:23]: It's a typo.

**Review by guihecheng — APPROVED** [review]
LGTM+1

**Pull request merged**